pkg/obi/config.go

```diff
@@ -252,6 +257,10 @@ type HostIDConfig struct {
 	FetchTimeout time.Duration `yaml:"fetch_timeout" env:"OTEL_EBPF_HOST_ID_FETCH_TIMEOUT"`
 }
 
+type NodeJsConfig struct {
```

Comment:
nit: Can we name this using the capitalized initialism?

```suggestion
type NodeJSConfig struct {
```

Reply:
done :)